README.rst

```diff
@@ -1002,6 +1002,17 @@ Choose from the list of available rules:
 
   The correct case must be used for standard PHP types in phpdoc.
 
+* **phpdoc_types_order**
+
+  Sorts PHPDoc types.
+
+  Configuration options:
+
+  - ``null_position`` (``'always_first'``, ``'always_last'``, ``'default'``): forces the
+    position of ``null``; defaults to ``'always_first'``
+  - ``sort_algorithm`` (``'alpha'``, ``'none'``): the sorting algorithm to apply;
```

Comment:
Yeah I wanted to implement something like that, but now I'd rather get this one merged and implement the new sort algorithm in a separate PR as it would probably lead to long discussions :)